### PR TITLE
8410 define methods and consume events api

### PR DESCRIPTION
**Pull request by andrelashley**

### CODE REVIEW

File: PelotonEppSdk/Models/VariableType.cs [hunk 1]
```diff
@@ -0,0 +1,7 @@
+﻿namespace PelotonEppSdk.Models
+{
+    public enum VariableType
+    {
+        String = 0
+    }
+}
```
Comment (fraserd) [inline PelotonEppSdk/Models/VariableType.cs]: There is a directory for Enum PelotonEppSdk.Enums. Please move this enum there too.

File: PelotonEppSdk/Models/VariableType.cs [hunk 2]
```diff
@@ -0,0 +1,7 @@
+﻿namespace PelotonEppSdk.Models
+{
+    public enum VariableType
```
Comment (fraserd) [inline PelotonEppSdk/Models/VariableType.cs]: I gather that the VariableType enum is meant to be used with EventCustomField, but it is never used. Can you demonstrate its usage, perhaps with a unit test?

File: PelotonEppSdkTests/PelotonEppSdkTests.csproj [hunk 3]
```diff
@@ -56,6 +56,7 @@
     <Compile Include="CreditCardCreateAndTransactionAndDeleteTests.cs" />
     <Compile Include="CreditCardCreateTests.cs" />
     <Compile Include="CreditCardTransactionTokenRequestTests.cs" />
+    <Compile Include="EventsTest.cs" />
```
Comment (fraserd) [inline PelotonEppSdkTests/PelotonEppSdkTests.csproj]: To be consistent, this file should be called EventsTests.cs (Tests is plural.)

File: PelotonEppSdkTests/TestBase.cs [hunk 4]
```diff
@@ -10,7 +10,8 @@ public class TestBase
     {
         protected TestBase()
         {
-            baseUri = new Uri("https://testapi.peloton-technologies.com");
+            // baseUri = new Uri("https://testapi.peloton-technologies.com");
+            baseUri = new Uri("http://localhost:2590/api");
```
Comment (fraserd) [inline PelotonEppSdkTests/TestBase.cs]: Please retract this baseuri change. It is unwanted in the master branch.

File: PelotonEppSdk/Models/VariableType.cs (hunk 1 above)
Comment (cattiwill) [reply PelotonEppSdk/Models/VariableType.cs]: Done

File: PelotonEppSdkTests/PelotonEppSdkTests.csproj (hunk 3 above)
Comment (cattiwill) [reply PelotonEppSdkTests/PelotonEppSdkTests.csproj]: Done

**Pull request merged**